backend/src/models/vulnerability.ts

```diff
@@ -136,6 +136,12 @@ export class Vulnerability extends BaseEntity {
     date: Date;
   }[];
 
+  @Column({
+    type: 'jsonb',
+    default: []
+  })
+  exposedEmails: object[];
```

Comment:
Can we actually just store it as a generic field called `structuredData`? This way, we can use this field for other future scans that also store structured data such as emails. What do you think?

```suggestion
  structuredData: object;
```